tests/ident/test_engine_parser_msgfplus_2021_03_22.py

```diff
@@ -176,30 +135,22 @@ def test_engine_parsers_msgfplus_get_peptide_lookup():
             "omssa_mod_dir": Path(__file__).parent.parent / "data",
         },
     )
-    assert len(parser.peptide_lookup) == 24
-    assert "Pep_YICDNQDTISSK" in parser.peptide_lookup.keys()
-    assert parser.peptide_lookup["Pep_YICDNQDTISSK"]["Sequence"] == "YICDNQDTISSK"
-    assert parser.peptide_lookup["Pep_YICDNQDTISSK"]["Modifications"][0]["pos"] == "3"
-    assert (
-        parser.peptide_lookup["Pep_YICDNQDTISSK"]["Modifications"][0]["mass"]
-        == "57.021464"
-    )
-    assert (
-        parser.peptide_lookup["Pep_YICDNQDTISSK"]["Modifications"][0]["name"]
-        == "Carbamidomethyl"
-    )
+    lookup = parser._get_peptide_lookup()
+    assert len(lookup) == 24
+    assert "Pep_YICDNQDTISSK" in lookup.keys()
```

Comment:
not clear to me why there is Pep_ prefix, must read the _get_peptide_lookup code, I guess - or maybe you can explain ..

Reply:
Thats how its saved in the mzid, its basically a reference tag